spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewriteDataFilesProcedure.java

```diff
@@ -218,6 +218,44 @@ public void testRewriteDataFilesWithZOrder() {
     assertEquals("Should have expected rows", expectedRows, sql("SELECT * FROM %s", tableName));
   }
 
+  @Test
+  public void testRewriteDataFilesWithZOrderNullBinaryColumn() {
```

Comment:
I would probably skip the procedure tests. I think the Util test covers this. If you did want to add a deeper integration test I would add it in the Action but I don't think that's required.

Reply:
Makes sense, I went back and forth on the same thing, but then I went ahead with adding the procedure test just to make sure we're covered in case for whatever reason down the line, the procedure class stops using the Util class or something. I'll leave it as is for now.